Sources/Mocker.swift

```diff
@@ -27,7 +27,28 @@ public struct Mocker {
     private(set) var mocks: [Mock] = []
     
     /// URLs to ignore for mocking.
-    private(set) var ignoredURLs: [URL] = []
+    public var ignoredURLs: [URL] {
+        ignoredRules.map { $0.urlToIgnore }
+    }
+
+    private var ignoredRules: [IgnoredRule] = []
+
+    private struct IgnoredRule: Equatable {
```

Comment:
Can you move this struct to the top of this file? 

```swift
class Mocker {
    struct IgnoredRule {
```

Reply:
Sure I can 🙂